magprime/templates/emails/marketplace_auto_waitlisted.txt

```diff
@@ -9,7 +9,7 @@ that table.
 We also apologize because we were hoping to get this information out to applicants by Nov 8, 2015, 
 but due to a technical configuration error, this email was not sent until now.
 
-We'll be opening applications for the MAGFest 2017 Marketplace in summer 2016, so please try
+We'll be opening applications for the MAGFest 2017 Marketplace in summer {{ c.YEAR|add:1 }}, so please try
```

Comment:
This should probably just be ``{{ c.YEAR }}`` because ``c.YEAR`` is the year of the event (not the year of the current date), which in this case is 2016.  If we did year + 1, then we'd be saying that we were opening the 2017 Marketplace applications in the summer of 2017, which would be several months AFTER the next MAGFest.